recipes/cppunit/meta.yaml

```diff
@@ -0,0 +1,32 @@
+{% set name = "CppUnit" %}
+{% set version = "1.13.2" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: http://dev-www.libreoffice.org/src/{{ name|lower }}-{{ version }}.tar.gz
+  sha256: 3f47d246e3346f2ba4d7c9e882db3ad9ebd3fcbd2e8b732f946e0e3eeb9f429f
+
+build:
+  number: 0
+  skip: True  # [win]
+
+requirements:
+  build:
+  run:
+
+test:
+  commands:
+    - cppunit-config --version
+
+about:
+  home: https://freedesktop.org/wiki/Software/cppunit/
+  license: LGPL-2.1
```

Comment:
Is there a license file in the source that we can package? If so, please add it with [`license_file`]( https://conda.io/docs/building/meta-yaml.html#license-file ).